admin/app.js

```diff
@@ -22,11 +23,41 @@ const helmet = require('helmet')
 const config = require('./config')
 const devWhitelist = require('./whitelist-dev')
 const azure = require('./azure')
+const featureToggles = require('feature-toggles')
+const winston = require('winston')
+
+winston.info('ENVIRONMENT_NAME : ' + process.env.ENVIRONMENT_NAME)
+const environmentName = process.env.ENVIRONMENT_NAME
+
+/**
+ * Load feature toggles
+ */
+let featureTogglesSpecific, featureTogglesDefault
+let featureTogglesSpecificPath, featureTogglesDefaultPath
+try {
+  featureTogglesSpecificPath = './config/feature-toggles-' + environmentName
+  featureTogglesDefaultPath = './config/feature-toggles-default'
+  featureTogglesSpecific = environmentName ? require(featureTogglesSpecificPath) : null
+  featureTogglesDefault = require(featureTogglesDefaultPath)
+} catch (error) {
+}
+
+if (featureTogglesSpecific) {
+  winston.info(`Loading environment-specific feature toggles from '${featureTogglesSpecificPath}' : `, featureTogglesSpecific)
```

Comment:
please perform a merge with the defaults here.  OR does the package take care of that for you, by applying them in order?